packages/eslint-config-airbnb/rules/react.js

```diff
@@ -23,13 +23,18 @@ module.exports = {
         'getDefaultProps',
         'getChildContext',
         'componentWillMount',
+        'UNSAFE_componentWillMount',
         'componentDidMount',
         'componentWillReceiveProps',
+        'UNSAFE_componentWillReceiveProps',
         'shouldComponentUpdate',
         'componentWillUpdate',
+        'UNSAFE_componentWillUpdate',
         'componentDidUpdate',
         'componentWillUnmount',
         'componentDidCatch',
+        'getDerivedStateFromProps',
```

Comment:
This shouldn't be needed, because `getDerivedStateFromProps` is a static method - static methods aren't checked by this rule.